package.json

```diff
@@ -4,16 +4,20 @@
   "description": "DB containing GP profile information",
   "main": "app.js",
   "scripts": {
-    "git-hooks": "yarn run lint && yarn run test",
+    "check-coverage": "istanbul check-coverage --config .istanbul.yml",
+    "generate-coverage": "yarn run istanbul -- cover _mocha -- --recursive",
+    "git-hook": "yarn run lint && yarn run generate-coverage && yarn run check-coverage",
```

Comment:
Given the automated check for Coveralls worked OK, I guess not!
When running in containers the token needs passing through but here it is just running locally on Travis.